### ISSUE TITLE
How to include target first name/last name in "To:" field?

### ISSUE THREAD

**Issue by ecbftw** [gophish/gophish#649]
Hello,

Thanks for providing such a useful tool.

I would like to launch a campaign where the email headers look like this:

```
...
From: Evil Haxor <evil@example.com>
To: Targeted Victim <target@example.com>
...
```

I'm able to get the `From:` header to show what I want by just setting the From field in the Sending Profile to exactly that string.  However, I can't figure out how to get the `To:` header to appear as above.  All I ever get is a `To:` header with the email address appearing by itself.  This doesn't look quite as convincing to the target when doing spear phishing... =(

I've reviewed the documentation and searched all past tickets but I can't find any discussion of this.  Can you hit me with a clue stick?

Thanks!
tim

**Comment by ecbftw** [comment]
A bit of additional info: I just tried adding a `To` header to my Sender Profile which contained:
```
{{.FirstName}} {{.LastName}} <{{.Email}}>
```
But this didn't work.  The `To:` header in the message remained a simple email address and no extra header was added. =(  Same for the test email.  Using v0.3 64bit.

tim

**Comment by jordan-wright** [comment]
Hey there! Yeah, I'm afraid this isn't possible at the moment. But I like the idea.

I'll add in a patch that adds the first/last names to the To header. Good call!

**Comment by jordan-wright** [comment]
I added the `contributor-friendly` label since this _should_ be a relatively simple fix if anyone wants to try and get to it before me. The code is here https://github.com/gophish/gophish/blob/master/worker/worker.go#L171 and here https://github.com/gophish/gophish/blob/master/worker/worker.go#L305

**Comment by ecbftw** [comment]
Thanks @jordan-wright !  I just realize I may need to supply names in a few different formats, depending on the customer.  For instance, I may want to make it look like:
```
To: Victim, Targeted <target@example.com>
```
In some cases, instead of what I showed above.  I think *some* default format that includes the firstname & last name makes sense, but some way to customize it is also desirable.  Not sure the best way to go with those more complex cases...

**Comment by jordan-wright** [comment]
This was addressed in 269568148ef928c7bbfd34cc63a02996224ce762. Now if the first and last name are specified the email is sent in "First Last" <email@domain.com> format.

Thanks for reporting this!

**Comment by ecbftw** [comment]
Awesome!  Thanks so much for adding this.  And I just realized that if I really want to include the names in "{last}, {first}" format, then I can just set the first name to "{last}," and the last name to "{first}".  I'll try this out on my next test.
